packages/rollups-node/s6-overlay/stage2-hook.sh

```diff
@@ -1,8 +1,8 @@
 #!/command/with-contenv sh
+set -e
 
 # check whether to use host-runner or not
 if [ ! -z ${ENABLE_SUNODO_NODE+x} ]; then
-    /command/s6-echo "host-runner: ENABLE_SUNODO_NODE is set, running in host mode..."
```

Comment:
There is a related issue which is showing the host-runner is running and at what address.
I did not include it in the PR #231

Reply:
Should I fix this in this PR?